### PR TITLE
SC-14561 add fleet and discovery

### PR DESCRIPTION
**Pull request by cpipilas**
Integrate our new Fleet and Discovery in our docs.

Added / Changed:

- New Fleet and Discovery section. Subsections:
    - Overview - New page with info/links about both Fleet, Service and Logs Discovery
    - Fleet - Updated
- Under Monitoring, `Autodiscovery` page renamed to `Service Discovery` and content updated
- Under Logs, `Discovery` section updated
- `Agents > Sematext Agent > Autodiscovery` page renamed to `Service Discovery` a new Overview page added
- New `Fleet & Discovery` tile in docs Getting Started page

### CODE REVIEW

**Comment by otisg** [comment]
Haven't looked at the actual content yet, just scanned the TOC to see what was changed/added and one thing I wondered about is whether "Fleet & Discovery" as a separate top-level thing, while also having Monitoring > Autodiscovery and Logs > Discovery (btw are we doing to update those? content or screenshots or both?).  

Thinking about this *very* briefly now...thinking about this now out loud... I also wondered if F&D should go under User Guide given that it feels like a platform sort of functionality.  But then again we have things like Alerts, Events, Dashboards, and Teams, all of which are also platform-level things, but we exposed them at top-level for visibility.

Should F&D come after user Guide since F&D is what people see first when they join?
Should Team come after F&D because inviting is often done early during onboarding?

File: docs/fleet/fleet.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+title: Fleet Overview
+description: Overview of Fleet - Sematext Agent management panel
+
+The Fleet section of Fleet and Discovery gives you an overview of all the Sematext Agents that are running in your systems.
+
+![Fleet Screen](../images/fleet/fnd-agents-collapsed.png)
+
+At first glance, you can see the following data in the table:
+
+- Host: Names of the hosts where the Sematext Agent is installed. You can either click on the name to open the Agent Information Panel or click on the expand view to reveal the list of Discovered Services available on the specific host. For further details, refer to the individual sections below in the document.
```
Comment (otisg) [inline docs/fleet/fleet.md:10]: Names of the hosts ==> Names of hosts

> where the Sematext Agent is installed

Actually, it's more about where it's running, right?
Or where it was running within the last N hours?
Which makes me think of 2 things:

1. People sometimes ask about what N is.  The answer is that the host/agent will disappear from this list after a while.  We should document that N.  And maybe we can also just say that somewhere in the UI?
2. Installed vs. running.  You know how we have inventory > packages?  Imagine if we compared that to the list of hosts where STA is running.  We could have a screen, maybe under F&D or maybe elsewhere that shows the "diff".  Know what I mean?  Hosts where you have STA running, but it's not running.  We have the data.  Also have STA version data, so can label those that could be updated.  We could also link to docs where we show commands for "parallel STA start" and  "parallel STA update" on N hosts.  See what I mean? Mmmmmmm. Ticket material.  Straight forward UI and impl.

File: docs/fleet/fleet.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+### Diagnostics
+
+If you are having issues sending data from your host, in this tab you can find the most recent logs from the agent, as well as some of what's inside the configuration files. You can find our Agent Troubleshooting documentation [here](https://sematext.com/docs/agents/sematext-agent/agent-troubleshooting). If you need any help troubleshooting, you can click on the `Report a Problem` button, write a short description of your issue and our support team will receive the message alongside a copy of the diagnostics data.
+
+Remember you can always contact us via chat on the bottom right corner of [our webpage](https://sematext.com/) and the Help button on Sematext Cloud, or via email at [support@sematext.com](mailto:support@sematext.com).
+
+![Agent Diagnostics](../images/fleet/agent-diagnostics.png)
+
+### Upgrade Instructions
```
Comment (otisg) [inline docs/fleet/fleet.md:68]: Aha, this could be another place where we show "parallel update on N hosts" sort of command.

**Review by otisg — CHANGES_REQUESTED** [review]
I think I looked at everything.  Feels like juicy additions and changes.

File: docs/logs/discovery/setup.md [hunk 3]
```diff
@@ -3,6 +3,7 @@ description: Getting started with logs discovery
 
 To set up logs discovery simply deploy the Sematext Agent in your infrastructure. Head to the [installation](/agents/sematext-agent/installation) instructions to proceed with the deployment. You'll have to deploy the Sematext Agent on every host where you would wish to discover and manage log files and container logs.
 
-Once you have successfully deployed the agent, navigate to the `Discovery > Logs` screen where you can explore all of your discovered log sources.
+Once you have successfully deployed the agent, navigate to the `Fleed & Discovery > Discovery > Logs` screen where you can explore all of your discovered log sources.
```
[the remark this answers is not in the document]
Comment (cpipilas) [reply docs/logs/discovery/setup.md:6]: That was hard to find!

File: docs/images/fleet/fnd-discovery-services-connected-apps.png
[the remark this answers is not in the document]
Comment (cpipilas) [reply docs/images/fleet/fnd-discovery-services-connected-apps.png:1]: This has been implemented.

File: docs/agents/sematext-agent/autodisco/index.md [hunk 4]
```diff
@@ -1,9 +1,11 @@
 title: Sematext Agent Service Discovery
 description:   Service Discovery: Additional Configuration for Sematext Agent
 
-For special cases, such as defining different sets of credentials in Kubernetes, additional steps are required to properly configure [service discovery](https://sematext.com/docs/monitoring/autodiscovery/).
+[Automatic Service Discovery](https://sematext.com/docs/monitoring/autodiscovery/) and Monitoring is a key feature of [Sematext Cloud](https://sematext.com/cloud/). The Sematext Agent contains most of the key components needed to make service discovery work. In the majority of the cases, everything is taken care automatically by the agent. However, for special cases, such as defining different sets of credentials in Kubernetes, additional steps are required to properly configure service discovery, as shown below:
```
Comment (otisg) [inline docs/agents/sematext-agent/autodisco/index.md:4]: taken care ==> taken care of

File: docs/agents/sematext-agent/autodisco/index.md [hunk 5]
```diff
@@ -1,9 +1,11 @@
...
 
 - Defining monitoring agent [credential sets](./credential-sets) in Kubernetes
 - Providing [MySQL JDBC driver](./mysql-driver) in container environments
 - [JMX Attaching](https://sematext.com/docs/agents/sematext-agent/jmx-attaching/)
 - [Removing stale](https://sematext.com/docs/agents/sematext-agent/autodisco/removing-stale-resources/) Sematext Agent resources after deinstallation
+
+For more information about service discovery in Sematext, please refer to our [Fleet & Discovery](https://sematext.com/docs/fleet/) section.
```
Comment (otisg) [inline docs/agents/sematext-agent/autodisco/index.md:11]: refer to our ==> refer to the
section ==> docs

File: docs/fleet/fleet.md [hunk 6]
```diff
@@ -7,7 +7,7 @@ The Fleet section of Fleet and Discovery gives you an overview of all the Semate
 
 At first glance, you can see the following data in the table:
 
-- Host: Names of the hosts where the Sematext Agent is installed. You can either click on the name to open the Agent Information Panel or click on the expand view to reveal the list of Discovered Services available on the specific host. For further details, refer to the individual sections below in the document.
+- Host: Names of hosts where the Sematext Agent is installed and running. You can either click on the name to open the Agent Information Panel or click on the expand view to reveal the list of Discovered Services available on the specific host. For further details, refer to the individual sections below in the document.
```
Comment (otisg) [inline docs/fleet/fleet.md:10]: Agent Information Panel looks like the name of something one should see in the UI.  Maybe this is referring to some Info tab?
list of Discovered Services ==> list of discovered services
available on the specific host ==> running on the specific host
individual sections below in the document ==> individual sections below

**Review by otisg — APPROVED** [review]
Just a few minor wording tweaks suggested.

File: docs/fleet/fleet.md (hunk 6 above)
Comment (cpipilas) [reply docs/fleet/fleet.md:10]: > Agent Information Panel 
It refers to the full flyout shown when one clicks a hostname in Fleet

**Pull request merged**